src/main/java/org/folio/inventory/resources/ItemRepresentation.java

```diff
@@ -109,7 +109,7 @@ private JsonObject toJson(
       contributorName.put("name", ((JsonObject)contributor).getString("name"));
       contributorNames.add(contributorName);
     });
-
+    representation.put(Item.ADMINISTRATIVE_NOTES_KEY, item.getAdministrativeNotes());
```

Comment:
Because the storage module always returns the record with the field present, even if it's blank.  Other notes fields, like circulation notes and notes, work the same way and use put instead of includeifpresent.

Reply:
Ok, I wasn't aware of that